### DISCUSSION TITLE
Cross Platform Development

### DISCUSSION THREAD

**Discussion by cbwolfe94** [k-paul-acct/algorithms-and-data-structures#2] [Ideas]
I use Linux, so can we move development to something like VS Code so we can have something consistent to work with? Visual Studio is not available on Linux since it's a Microsoft product only.

**Comment by k-paul-acct** [comment]
I also use Linux and I write code in JetBrains Rider) VS Code can be a good alternative, but without much code analysis that Rider performs.

**Reply by cbwolfe94** [reply]
What kind of code analysis does it do?

**Reply by k-paul-acct** [reply]
Some static analysis, quick fixes issues. Just take a look, that's great!
https://www.jetbrains.com/help/rider/Code_Analysis__Index.html

**Reply by cbwolfe94** [reply]
> Some static analysis, quick fixes issues. Just take a look, that's great! https://www.jetbrains.com/help/rider/Code_Analysis__Index.html

There's most likely a plug in that can do static analysis for C#
